### PR TITLE
BUG and DOC: fixed tools.analyse.info in all examples and added (C) pl…

### PR DESCRIPTION
**Pull request by govarguz**
BUG and DOC: fixed tools.analyse.info in all examples and added (C) to all of them in an homogeneous manner; also renamed a file that was referring to the old espresso ext

### CODE REVIEW

File: examples/parallel_tempering/parallel_tempering.py [hunk 1]
```diff
@@ -7,7 +31,7 @@
 
 print "+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
 print "+ Due to a recent change in the design of ESPResSo++ the Parallel Tempering     +"
-print "+ Class is not available in the current version. This Class will be back soon.  +"
```
Comment (MrTheodor) [inline examples/parallel_tempering/parallel_tempering.py]: If so, does it make sense to print this whole statement?

File: examples/adress/fadress_protein/ubiquitin.py [hunk 2]
```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env python2
```
Comment (junghans) [inline examples/adress/fadress_protein/ubiquitin.py]: Why dropping the shebang?

File: examples/adress/hadress_tetraliquid/hadress_tetraliquid_KTI/hadressKTI.py [hunk 3]
```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env python2
-
-#  Copyright (C) 2016
+#  Copyright (C) 2017
```
Comment (junghans) [inline examples/adress/hadress_tetraliquid/hadress_tetraliquid_KTI/hadressKTI.py]: 2016-2017, I guess!

File: examples/parallel_tempering/parallel_tempering.py [hunk 4]
```diff
@@ -7,7 +31,7 @@
...
+print "+ Class is available in the current version.                                    +"
```
Comment (junghans) [inline examples/parallel_tempering/parallel_tempering.py]: Just drop that line!

File: examples/adress/hadress_tetraliquid/hadress_tetraliquid_FEC/Tetracryst.py [hunk 5]
```diff
@@ -1,4 +1,21 @@
-#!/usr/bin/python
```
Comment (junghans) [inline examples/adress/hadress_tetraliquid/hadress_tetraliquid_FEC/Tetracryst.py]: Keep!

**Pull request merged**